### ISSUE TITLE
Plone 4.x, spinner.gif image should have alt text.

### ISSUE THREAD

**Issue by ivanteoh** [plone/Products.CMFPlone#1526]
Recently one of our 4.x plone sites was audited for WCAG 2.0 AA. The report came back and mentioned that `spinner.gif` image should have alt text because this image does have a specific purpose during ajax call loading.

```
<div style="display: none;" id="ajax-spinner">
<img src="http://localhost:8080/Plone/spinner.gif" alt="">
</div>
```

**Comment by jensens** [comment]
I close the issue, because it addresses a Plone version that is no longer supported. 
If you think this is wrong please reopen the issue and assign a matching milestone.
